redisson/src/test/java/org/redisson/executor/RedissonExecutorServiceTest.java

```diff
@@ -670,4 +671,27 @@ public void run() {
         });
     }
 
+    @Test
+    public void testTaskDelay4TaskService() throws IllegalAccessException, NoSuchFieldException, InterruptedException {
+        RScheduledExecutorService test = redisson.getExecutorService("test");
+        Field field = RedissonExecutorService.class.getDeclaredField("schedulerChannelName");
```

Comment:
"Reflect.on("... should be used. You can find its usage in org.redisson.RedissonMapCacheTest#testRemoveEmptyEvictionTask()